package.json

```diff
@@ -4,17 +4,22 @@
   "license": "UNLICENSED",
   "type": "module",
   "scripts": {
-    "install_all": "(cd frontend && yarn install) && (cd backend && yarn install) && (cd shared && yarn install)",
+    "install_all": "yarn install && (cd frontend && yarn install) && (cd backend && yarn install) && (cd shared && yarn install)",
     "build": "(cd ./frontend && yarn build)",
     "start": "(cd frontend && yarn start)",
     "dev": "(cd frontend && yarn dev)",
-    "update_test_data": "export $(grep -v '^#' backend/.env.development.local | xargs) && BACKEND_FOLDER=backend/ ./backend/test_helpers/updateTestData.sh",
+    "drop_actions_and_tasks_and_types": "yarn ts-node --esm ./backend/dropActionsAndTasksAndTypes.ts",
+    "update_local_seed_file": "export $(grep -v '^#' backend/.env.development.local | xargs) && PREPEND_BACKEND_SUBFOLDER=true ./backend/data/updateLocalSeedFile.sh",
+    "update_test_data": "export $(grep -v '^#' backend/.env.development.local | xargs) && PREPEND_BACKEND_SUBFOLDER=true ./backend/test_helpers/updateTestData.sh",
+    "seed": "export $(grep -v '^#' backend/.env.development.local | xargs) && PREPEND_BACKEND_SUBFOLDER=true ./backend/data/seedLocalDatabase.sh",
     "test": "DOTENV_CONFIG_PATH=.env.test jest",
     "t": "(cd backend && DOTENV_CONFIG_PATH=.env.test jest --testNamePattern=\"_tx_\" src/helpers/updateTransactions.test.ts)",
```

Comment:
Didn't notice before, what is `yarn t` for? I didn't find it in the documentation

Reply:
Good catch. I'm not sure that I will even keep this command in here because I just made it as a temporary easy-to-type filtered test command that runs just a subset of tests. I might end up keeping it (and renaming it and documenting it) if it continues to be useful.